lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -739,10 +791,7 @@ namespace Js
         EngineInterfaceObject_CommonFunctionProlog(function, callInfo);
         INTL_CHECK_ARGS(args.Info.Count == 2 && JavascriptString::Is(args.Values[1]));
 
-        JavascriptString *locale = JavascriptString::UnsafeFromVar(args.Values[1]);
-        utf8::WideToNarrow locale8(locale->GetSz(), locale->GetLength());
-
-        return TO_JSBOOL(scriptContext, findLocale(locale8, udat_countAvailable, udat_getAvailable));
+        return TO_JSBOOL(scriptContext, findLocale(JavascriptString::UnsafeFromVar(args.Values[1]), udat_countAvailable, udat_getAvailable));
```

Comment:
>JavascriptString::UnsafeFromVar(args.Values[1]) [](start = 51, length = 47)

nit: I think I'd still prefer this to be a local var to make this read a bit easier.